pkg/cmd/server/kubernetes/master/controller/volumes.go

```diff
@@ -53,47 +45,23 @@ func (c *PersistentVolumeControllerConfig) RunController(ctx kubecontroller.Cont
 		return false, err
 	}
 
-	volumeController := persistentvolumecontroller.NewController(
+	volumeController, volumeControllerErr := persistentvolumecontroller.NewController(
 		persistentvolumecontroller.ControllerParameters{
 			KubeClient:                ctx.ClientBuilder.ClientOrDie("persistent-volume-binder"),
 			SyncPeriod:                ctx.Options.PVClaimBinderSyncPeriod.Duration,
-			AlphaProvisioner:          alphaProvisioner,
 			VolumePlugins:             plugins,
-			Cloud:                     c.CloudProvider,
+			Cloud:                     ctx.Cloud,
 			ClusterName:               ctx.Options.ClusterName,
 			VolumeInformer:            ctx.InformerFactory.Core().V1().PersistentVolumes(),
 			ClaimInformer:             ctx.InformerFactory.Core().V1().PersistentVolumeClaims(),
-			ClassInformer:             ctx.InformerFactory.Storage().V1beta1().StorageClasses(),
+			ClassInformer:             ctx.InformerFactory.Storage().V1().StorageClasses(),
 			EventRecorder:             recorder,
 			EnableDynamicProvisioning: ctx.Options.VolumeConfiguration.EnableDynamicProvisioning,
 		})
-
-	go volumeController.Run(ctx.Stop)
-
-	return true, nil
-}
-
-type PersistentVolumeAttachDetachControllerConfig struct {
```

Comment:
got replaced by the "normal" one from upstream: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-controller-manager/app/controllermanager.go#L332-L333